common/src/main/java/oracle/kubernetes/common/utils/SchemaConversionUtils.java

```diff
@@ -163,6 +166,7 @@ private void convertDomainStatus(Map<String, Object> domain) {
     } else { // 9 or above
       removeObsoleteConditionsFromDomainStatus(domain);
       removeUnsupportedDomainStatusConditionReasons(domain);
+      renameServerStatusFields(domain);
```

Comment:
Do you have convert to OLD_STATE_GOAL_KEY  from NEW_STATE_GOAL_KEY to provide fidelity from v9 to v8?

Reply:
Thanks. I'll add that.